src/main/java/org/sonarlint/intellij/cayc/CleanAsYouCodeService.kt

```diff
@@ -36,11 +37,20 @@ class CleanAsYouCodeService {
 
     fun setFocusOnNewCode(isFocusOnNewCode: Boolean) {
         getGlobalSettings().isFocusOnNewCode = isFocusOnNewCode
+        refresh()
```

Comment:
Last suggestion: this first method could call the second by providing getGlobalSettings() as the parameter.